### PR TITLE
Change the URL of TizenFX API Reference

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

Change the URL of TizenFX API Reference to https://samsung.github.io/TizenFX/latest/api/

### CODE REVIEW

**Comment by mijin85cho** [comment]
One big question before the detailed review comment:

There are about 10 .NET sample apps with links to "https://samsung.github.io/TizenFX/API5~", but only 3 .NET sample apps are fixed. 

Is it sure that only 3 .NET sample application are updated in this PR? Would you please check?

File: docs/application/dotnet/guides/multimedia/stream-recorder.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ The main features of the `Tizen.Multimedia.StreamRecorder` class include:
         -   Video: MP4 and 3GP
         -   Audio: AMR, AAC, and WAV
 
-        The supported file formats are defined in the [Tizen.Multimedia.RecorderFileFormat](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.RecorderFileFormat.html) enumeration.
+        The supported file formats are defined in the [Tizen.Multimedia.RecorderFileFormat](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.RecorderFileFormat.html) enumeration.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/multimedia/stream-recorder.md]: fixed.

File: docs/application/dotnet/guides/multimedia/thumbnail-images.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The requested thumbnail is provided as a raw data type with the BGRA color space
 
 ## Prerequisites
 
-To use the methods and properties of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class, include the [Tizen.Multimedia.Util](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Util.html) namespace in your application:
+To use the methods and properties of the [Tizen.Multimedia.Util.ThumbnailExtractor](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class, include the [Tizen.Multimedia.Util](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.html) namespace in your application:
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/multimedia/thumbnail-images.md]: fixed.

File: docs/application/dotnet/guides/nlp/nlp.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ The main features of the Tizen.Nlp namespace include:
 
 To enable your application to use the NLP functionality:
 
-1.  To use the [Tizen.Nlp](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Nlp.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+1.  To use the [Tizen.Nlp](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.Nlp.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/nlp/nlp.md]: fixed.

File: docs/application/dotnet/guides/notification/attach-panel.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ The following figure explains the content types. From left to right: images, cam
 
 This section explains, how you can enable your application to use the attach panel functionality:
 
-1.  To use the [Tizen.Applications.AttachPanel](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.AttachPanel.html) namespace, the application must request permission by adding the following privileges to the `tizen-manifest.xml` file:
+1.  To use the [Tizen.Applications.AttachPanel](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.Applications.AttachPanel.html) namespace, the application must request permission by adding the following privileges to the `tizen-manifest.xml` file:
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/notification/attach-panel.md]: fixed.

File: docs/application/dotnet/guides/notification/noti.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ The main features of the Tizen.Applications.Notifications namespace include:
     -   You can [set styles](#style) for a notification, changing its [display type](#type) and adding features, such as icons and background images.
 -   Posting a notification
 
-    You can [post a notification](#post) using the `Post()` method of the [Tizen.Applications.Notifications.NotificationManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Notifications.NotificationManager.html) class.
+    You can [post a notification](#post) using the `Post()` method of the [Tizen.Applications.Notifications.NotificationManager](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.Applications.Notifications.NotificationManager.html) class.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/notification/noti.md]: fixed.

File: docs/application/dotnet/guides/nui/animation.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ _animation = new Animation
 
 Some view properties are `animatable`, such as `Position`, `Orientation`, `Scale`, and `Color`.
 
-The controls class is derived from the [Tizen.NUI.BaseComponents.CustomView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.BaseComponents.CustomView.html) class and set the property to be *animatable*. For standard controls, you can query whether a property is animatable (using the `IsPropertyAnimatable()` method of the [Tizen.NUI.Animatable](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Animatable.html) class), but cannot change the animatable state.
+The controls class is derived from the [Tizen.NUI.BaseComponents.CustomView](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.CustomView.html) class and set the property to be *animatable*. For standard controls, you can query whether a property is animatable (using the `IsPropertyAnimatable()` method of the [Tizen.NUI.Animatable](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Animatable.html) class), but cannot change the animatable state.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/nui/animation.md]: fixed.

File: docs/application/dotnet/guides/nui/button.md [hunk 7]
```diff
@@ -1,8 +1,8 @@
 # Button
 
-A button is a NUI control that you can press in order to operate it. You can create push buttons, check box buttons, radio buttons, and toggle buttons. All different buttons are implemented through the [Tizen.NUI.UIComponents.Button](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.UIComponents.Button.html) base class.
+A button is a NUI control that you can press in order to operate it. You can create push buttons, check box buttons, radio buttons, and toggle buttons. All different buttons are implemented through the [Tizen.NUI.UIComponents.Button](hhttps://samsung.github.io/TizenFX/latest/api/Tizen.NUI.UIComponents.Button.html) base class.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/nui/button.md]: fixed.

File: docs/application/dotnet/guides/personal/account.md [hunk 8]
```diff
@@ -493,7 +493,7 @@ The following table lists the properties that can be defined for each account pr
...
...count/capability/contact"`<br>If you register this capability in the manifest file, the user using the contact application can see a list of accounts with the account of your service in the contact application. <br>-Calendar capability:<br>	    `CalendarCapability` or `"http://tizen.org/account/capability/calendar"`<br>If you register this capability in the manifest file, the user using the calendar application can see a list of accounts with the account of your service in the calendar application.<br>-Email capability:<br>	    `EmailCapability` or `"http://tizen.org/account/capability/email"`<br>-Photo capability:<br>	    `PhotoCapability` or `"http://tizen.org/account/capability/photo"`<br>-Video capability:<br>	    `VideoCapability` or `"http://tizen.org/account/capability/video"`<br>-Music capability:<br>	    `MusicCapability` or `"http://tizen.org/account/capability/music" ` <br>-Document capability:<br>	    `DocumentCapability` or `"http://tizen.org/account/capability/document"` <br>-Message capability:<br>	    `MessageCapability` or `"http://tizen.org/account/capability/message"` <br>-Game capability:<br>	    `GameCapability` or `"http://tizen.org/account/capability/game"` |
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/personal/account.md]: fixed.

File: docs/application/dotnet/guides/personal/app-history.md [hunk 9]
```diff
@@ -7,18 +7,18 @@ The main features of the Tizen.Context.AppHistory namespace are:
 
 -   Retrieving application usage statistics
 
-    You can [retrieve application launch history](#retrieve_usage_stats), such as frequently used applications and recently used applications, using the [Tizen.Context.AppHistory.UsageStatistics](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Context.AppHistory.UsageStatisticsData.html) class.
+    You can [retrieve application launch history](#retrieve_usage_stats), such as frequently used applications and recently used applications, using the [Tizen.Context.AppHistory.UsageStatistics](https://samsung.github.io/TizenFX/latest/api/Tizen.Context.AppHistory.UsageStatisticsData.html) class.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/guides/personal/app-history.md]: fixed.

File: docs/application/dotnet/samples/multimedia/overview.md [hunk 10]
```diff
@@ -124,36 +124,36 @@ for TD:
                         <td><img alt="" height="180" src="media/w64mediacodecsample.png" width="180"/></td>
                         <td>
                         <p><strong>(W) MediaCodecSample</strong> [In progress]</p>
-                        <p>This sample application demonstrates how to encode audio and video data using <a href="https:/samsung.github.io/TizenFX/API5/api/Tizen.Multimedia.MediaCodec.html" target="_blank">Tizen.Multimedia.MediaCodec</a>.</p>
+                        <p>This sample application demonstrates how to encode audio and video data using <a href="https:/samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.MediaCodec.html" target="_blank">Tizen.Multimedia.MediaCodec</a>.</p>
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/dotnet/samples/multimedia/overview.md]: fixed.

**Pull request merged**